packages/default.nix

```
@@ -65,10 +65,10 @@
         // optionalAttrs isLinux {
           folder-size-metrics = pkgs.callPackage ./folder-size-metrics { };
         }
-        // optionalAttrs (system == "x86_64-linux") {
+        // optionalAttrs (system == "x86_64-linux") rec {
```

I don't think we need to make this attrset recursive:

```suggestion
        // optionalAttrs (system == "x86_64-linux") {
```